README.md

```diff
@@ -49,6 +49,16 @@ npm run preview-release
 # http://htmlpreview.github.io/?https://github.com/underdogio/underdogio.github.io/blob/{{preview_branch}}/index.html
 ```
 
+#### OSX Troubleshooting
+If you are on OSX and running into issues (e.g. `globstar` isn't defined and `sed` doesn't support `--in-place`), then please upgrade your `bash` and install `gnu-sed`:
+
+- `bash` upgrade available via shellshock test sites
```

Comment:
oh.. you know. I don't use `bash` as my shell, I use `zsh`. 

When doing `brew install bash` I get the desired results from `/usr/bin/env bash`. I cannot say how it works with `chsh` or if that is really desired.